### PR TITLE
500 create developer onboarding doc

### PR DESCRIPTION
**Pull request by mrzta**

### CODE REVIEW

File: documentation/developer-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+- There are 3 s189 subscriptions, one for each environment (dev, test and prod)
+
+  `s189-teacher-services-cloud-development`
+
+  `s189-teacher-services-cloud-test`
+
+  `s189-teacher-services-cloud-production`
+
+
+- There is an assumption that you have have been given a cip account
```
Comment (smneal) [inline documentation/developer-onboarding.md:15]: Delete the second `have`.

File: documentation/developer-onboarding.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+ *`Deployment`*: is a Kubernetes object that allows you to specify the desired state of your application. It allows you to deploy multiple pods and services and manage them as a single entity. It also allows you to do rolling updates and rollbacks.
+
+ *`Pods`*: are ephemeral and can be deleted or recreated at any time. Deployments provide a way to keep pods running and provide a way to update them when needed.
+
+ *Top pods*: `kubectl top pods` displays the CPU and memory usage of the pods in the current namespace. If you want to see the usage of pods in a specific namespace, you can use the -n flag.
+
+*You will need PIM to perform below actions*
+
+- To get deployments in test cluster you can use `kubectl get deployments --all-namespaces` this return all namespaces in the current cluster like below. If you havent done it already get your credentials to test cluster using `make test get-cluster-credentials CONFIRM_TEST=yes` this will set your cluster context as well.
```
Comment (smneal) [inline documentation/developer-onboarding.md:66]: `in **the** test`
`this **will** return`
`to **the** test cluster`

Comment (mrzta) [reply documentation/developer-onboarding.md]: removed the entire paragraph

**Review by nickdevdfe — APPROVED** [review]
Reviewed and approved.

**Pull request merged**